### ISSUE TITLE
Disable Satellites in Stellarium Web

### ISSUE THREAD

**Issue by neilparker62** [Stellarium/stellarium-web-engine#185]
Is it  possible to disable satellite view ?  There was an online video I saw which showed a second set of menu icions at the lower edge of the screen. I don't see those in the Stellarium web interface ?

**Comment by 31SFX4** [comment]
As there are so many satellites now, this feature is really necessary, otherwise we'll soon not be able to see any stars.

Does the rendering of the fast moving satellites consume much CPU?

**Comment by gzotti** [comment]
Not the rendering, but of course the computation takes a few cycles. And yes, the upcoming tsunamis of tens of thousands of satellites may slow down the program.

**Comment by judovana** [comment]
The ability to disabld satelites is indeed necesary. It is awesome as they are implemented, but few mote thousnds (year?) And they  will be disturbong more then enjoyable. Thanx a lot for awesome work

**Comment by gzotti** [comment]
Yes, satellite constellations are more disturbing than enjoyable. Especially on the real sky.

**Comment by Bill-J142** [comment]
Looks like it's been well over a year since this issue was brought up, and no action?  It truly sucks to have so many satellites drifting across the field of view. I am surprised there is not a bigger outcry.  Is it that complicated to offer users a switch to turn it off? Please?

**Comment by gzotti** [comment]
Yes, please write to Mr. Musk to stop destroying the sky, his activities are undesirable. Wide-field astrophotography becomes impossible without his space junk cutting lines everywhere.

**Comment by Bill-J142** [comment]
> Yes, please write to Mr. Musk to stop destroying the sky, his activities are undesirable. Wide-field astrophotography becomes impossible without his space junk cutting lines everywhere.

Amen. I am just a casual sky watcher but still appalled at what;s happened to the near-earth environment.  Stellarium drives home the reality of it, and that's good, but without a way to switch off, it has become depressing and just reminds me of Mr. Starcreep.  
I'm trying out TheSkyLive. It lacks Stellarium's fancier features but at least does not show artificial objects.

**Comment by ecorm** [comment]
To the maintainers of this project: would a PR that addresses this be approved if it passes review? I'm a professional C++ dev and I'm considering rolling up my sleeves and fixing this myself, but I don't want to waste my time if a PR is going to be ignored. I see that there are PRs going all the way back 2021 that are still open.

I find stellarium-web to be a useful resource when the astro-curious ask me for help in finding things in the night sky. I think it deserves some love.
